pimd/pim_zlookup.c

```diff
@@ -259,19 +267,15 @@ static int zclient_read_nexthop(struct pim_instance *pim,
 #else
 			/* RFC 5549 v4-over-v6 nexthop handling */
 
-			/*
-			 * If we are sending v6 secondary assume we receive v6
+			/* If we are sending v6 secondary assume we receive v6
```

Comment:
yeah whitespace changes in comments are not what we want to be doing

Reply:
Reverted unnecessary changes.